ui/admin/tests/acceptance/targets/create-test.js

```diff
@@ -99,9 +117,22 @@ module('Acceptance | targets | create', function (hooks) {
       )
     );
     assert.notOk(find(`[href="${urls.newTCPTarget}"]`));
+    assert.notOk(find(`[href="${urls.newSSHTarget}"]`));
+  });
+
+  test('cannot navigate to new SSH targets route when ssh feature is disabled', async function (assert) {
```

Comment:
This is where I am disabling the `ssh-target` feature flag that I would appreciate feedback on.

Reply:
We started a discussion about this in Slack. As per that discussion I am approving this PR.